draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -186,7 +186,11 @@ The following rules apply to validating the client attestation JWT. Application
 
 6. The JWT MAY contain an "iat" (issued at) claim that identifies the time at which the JWT was issued.
 
+<<<<<<< HEAD:draft-looker-oauth-attestation-based-client-auth.md
 7. The JWT MAY contain a "jti" (JWT ID) claim that provides a unique identifier for the token. The authorization server MAY ensure that JWTs are not replayed by maintaining the set of used "jti" values for the length of time for which the JWT would be considered valid based on the applicable "exp" instant.
+=======
+7. The JWT MAY contain a "jti" (JWT ID) claim that provides a unique identifier for the token.
+>>>>>>> main:draft-ietf-oauth-attestation-based-client-auth.md
```

Comment:
git merge conflict here, I'm in favor of the first sentence :-)
then we should put a ref to RFC7519, related to the section where the jti claim is defined